app/bot.py

```diff
@@ -38,23 +38,36 @@
 
 
 def send_form(data:Dict):
-	""" Sends form to user, inserts to conversations table """
+	""" Sends form to user in tweet or starts bot conversation based on form,
+	inserts to conversations table """
 
 	to_insert = DB.convert_invocation_conversations(data)
+	incident_id = to_insert['incident_id']
 	user_id_str = twitter.get_user_id_from_tweet(to_insert['tweet_id'])
+	form_link = f'https://a.humanrightsfirst.dev/edit/{incident_id}'
 	to_insert['tweet_id'] = int(to_insert['tweet_id'])
-	to_insert['tweet_text'] = '@' + to_insert['tweeter_id'] + ' ' + conversation_tree[10] + '\n' + dm_link 
-	to_insert['reachout_template'] = conversation_tree[10]
 	to_insert['in_reply_to_id'] = to_insert['tweeter_id']
 	del to_insert['link']
-
-
-	status = twitter.respond_to_tweet(to_insert['tweet_id'], to_insert['tweet_text'])
-	to_insert['tweeter_id'] = bot_name
```

Comment:
I can see where we are removing redundant code here to match the the changes you've made regarding the split.